### ISSUE TITLE
SDI output using Blackmagic hardware

### ISSUE THREAD

**Issue by canoemoose** [Alteka/Kards#40]
Three requests and counting, including one comment from a dev who's used the Macadam library before.

**Comment by DrewLX** [comment]
Yeah, Macadam is a decent library. It's making the data flow sensible from electron browser window that's harder. Holographics does this somehow though.

**Comment by canoemoose** [comment]
Yes, this seems as though it should be possible, unlike the NDI. The guy on Reddit mentioned making a PR "in the future, when he had time" but I said we'd have a think and reach out to him.

**Comment by DrewLX** [comment]
I think it would be worth reaching out and seeing what he knows about browser window > macadam. The rest of it is pretty straight forward. But Macadam requires a frame of RGBA data, and I'm not sure how you'd capture that at a decent rate, and pass it around the system.

**Comment by Coketech** [comment]
This would be excellent! We love Kards, output onto SDI would be next level helpful

**Comment by DrewLX** [comment]
The latest thoughts are we might be able to take inspiration from Vingester but concern is that we might not achieve reliable frame rates using macadam, whereas HDMI and a converter would work reliably. 

As a workaround for now Chris, you can use kards.alteka.solutions inside OBS and output to SDI that way. Not ideal but it does work!

**Comment by DrewLX** [comment]
Once we're updated to electron 16 I'll revisit this request. Taking the same virtual window buffer that Grandiose needs we could pipe the same thing to Macadam to drive Blackmagic cards.

**Comment by M0GUR** [comment]
Any further thoughts on this? It really would be great, I've now been tripped up a few times going down the HDMI to SDI adapter route.
